metadata.yml

```diff
@@ -14,9 +14,63 @@ annotations:
   io.kubewarden.policy.source: https://github.com/kubewarden/verify-image-signatures
   io.kubewarden.policy.license: Apache-2.0
   io.kubewarden.policy.usage: |
-    This policy validates Sigstore signatures for containers, init container and ephemeral container that match the name provided
-    in the `image` settings field. It will reject the Pod if any validation fails.
-    If all signature validation pass or there is no container that matches the image name, the Pod will be accepted.
+  This policy validates Sigstore signatures for containers, init container and ephemeral container that match the name provided
```

Comment:
This is indented wrong, making that invalid YAML. You should add one more indentation level starting from this line